response_operations_ui/error_handlers.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from requests import RequestException
+from structlog import wrap_logger
+
+from response_operations_ui import app
+from response_operations_ui.exceptions.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@app.errorhandler(ApiError)
+def api_error(error):
+    logger.error('Api failed to retrieve required data',
+                 url=error.url, status_code=str(error.status_code))
+    return "FAIL"
```

Comment:
Might be cleaner to return a 500 status code here `return Response(status_code=500)`

Reply:
The "FAIL" text is just a placeholder for actual error pages which we'll add in later stories